src/Utils.cpp

```diff
@@ -156,3 +164,43 @@ size_t Utils::ceilMultiple(size_t a, size_t b) {
     auto ret = a + (b - a % b);
     return ret;
 }
+
+#ifdef _WIN32
+// https://stackoverflow.com/a/3999597
+// Convert a wide Unicode string to an UTF8 string
+// Thanks Windows, saving developers time since 1995
+std::string utf8_encode(const std::wstring &wstr)
+{
+    if ( wstr.empty() ) return std::string();
+    int size_needed = WideCharToMultiByte(CP_UTF8, 0, &wstr[0], (int)wstr.size(), NULL, 0, NULL, NULL);
+    std::string strTo(size_needed, 0);
+    WideCharToMultiByte(CP_UTF8, 0, &wstr[0], (int)wstr.size(), &strTo[0], size_needed, NULL, NULL);
+    return strTo;
+}
+#endif
+
+std::string Utils::leelaz_file(std::string file) {
+#ifdef _WIN32
+    // https://stackoverflow.com/a/11032792 but avoiding the UNICODE mess and just using W variants
+    wchar_t szPath[MAX_PATH];
+    if (!SUCCEEDED(SHGetFolderPathW(NULL, CSIDL_COMMON_APPDATA, NULL, 0, szPath)))
```

Comment:
Given that we have no installer who plunks things into AppData etc, there's no reason to assume the network will be there. For Windows it's fine to just keep the current behavior and avoid all this complication, I made remarks before because the Linux support was changing (and breaking) Windows. It doesn't necessarily need to do the same.

Reply:
Are you sure?

It does not change the current behaviour regarding `--weights`, it only avoids an error if no `--weights` is given and that file exists. However it does change the behaviour of where the `leelaz_opencl_tuning` file is saved, but arguably this is better even on Windows, since now all invocations of leelaz will write to one standard file rather than dropping these files all over the filesystem depending on what the user's CWD was when they launched leelaz.